scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1428,6 +1438,21 @@ def formPipelineJson(pipelineId, dbPipelineJson, stageJson):
         raise e
 
 
+
+def postingGateJsonSQL(pipelineJson):
+    try:
+        pipeline_id = pipelineJson['id']
+        update_data = json.dumps(pipelineJson), isd_admin_username, pipeline_id	
+        mysqlcursor = spindb_front50.cursor()
```

Comment:
I think having seperate cursor for fornt50 and orca db will look clean

Reply:
updated as suggested